pennylane/gradients/gradient_transform.py

```diff
@@ -312,13 +312,15 @@ def jacobian_wrapper(
                 if not isinstance(cjac, tuple):
                     is_square = cjac.ndim == 2 and cjac.shape[0] == cjac.shape[1]
 
-                    if is_square and qml.math.allclose(cjac, qml.numpy.eye(cjac.shape[0])):
-                        # Classical Jacobian is the identity. No classical processing
-                        # is present inside the QNode.
-                        return qjac
+                    if not qml.math.is_abstract(cjac):

```

Comment:
Why does `cjac` need to not be abstract before you check if it's an identity?

Reply:
Because you cannot access the values of jitted objects. therefore I have to skip it here otherwise it would fail for jit tests